.github/workflows/build_conda_pkg_workflow.yaml

```diff
@@ -21,19 +25,24 @@ jobs:
         with:
           auto-update-conda: true
           activate-environment: conda_pkg_build_env
-      - name: install conda-build and conda-verify
+      - name: install conda-build
         shell: bash -l {0}
         run: |
-          conda info
           conda install conda-build
+      - name: install conda-verify
+        shell: bash -l {0}
+        run: |
           conda install conda-verify
-          conda config --add channels conda-forge
+      - name: conda info
+        shell: bash -l {0}
+        run: |
+          conda info
       - name: Build pytango Conda package
         shell: bash -l {0}
         run: |
-          conda build -c tango-controls $GITHUB_WORKSPACE/pytango --output-folder=$GITHUB_WORKSPACE/dist
+          conda build -c main -c conda-forge -c tango-controls $GITHUB_WORKSPACE/pytango --variant-config-files $GITHUB_WORKSPACE/.github/conda_build_config_files/conda_build_config-${{ matrix.python_version }}.yaml --output-folder=$GITHUB_WORKSPACE/dist-${{ matrix.python_version }}
       - name: Upload pytango Conda Package as artifact
         uses: actions/upload-artifact@v2
         with:
-          name: pytango-linux-64-conda-package
-          path: dist/linux-64/pytango*.bz2
+          name: pytango-${{ matrix.python_version }}-linux-64-conda-package
```

Comment:
How do the artifacts we build here get uploaded to Anaconda Cloud?

I guess a future PR might add the Windows versions (previous builds needed the Windows wheels from PyPI, so we probably can't do that for v9.3.2, since those wheels are broken).

Reply:
> How do the artifacts we build here get uploaded to Anaconda Cloud?

This is currently a manual operation. (Simple anaconda upload command with an option to upload to tango-controls conda channel.

> I guess a future PR might add the Windows versions (previous builds needed the Windows wheels from PyPI, so we probably can't do that for v9.3.2, since those wheels are broken).

I guess we will have to do something for Windows too indeed. In another PR.